EpikCord/application.py

```diff
@@ -1,82 +1,59 @@
-        self.rpc_origins: Optional[List[str]] = data.get("rpc_origins")
-        self.bot_public: bool = data["bot_public"]
-        self.bot_require_code_grant: bool = data["bot_require_code_grant"]
-        self.terms_of_service_url: Optional[str] = data.get(
-            "terms_of_service_url"
-        )
-        self.privacy_policy_url: Optional[str] = data.get("privacy_policy_url")
+        self.id = int(data["id"])
+        self.name = data["name"]
+        self.icon = data.get("icon")
+        self.description = data["description"]
+        self.rpc_origins = data.get("rpc_origins")
+        self.bot_public = data["bot_public"]
+        self.bot_require_code_grant = data["bot_require_code_grant"]
+        self.terms_of_service_url = data.get("terms_of_service_url")
+        self.privacy_policy_url = data.get("privacy_policy_url")
         self.owner = data.get("owner")  # TODO: User Object
-        self.verify_key: str = data["verify_key"]
-        self.team: Optional[Team] = Team(data["team"]) if data["team"] else None
-        self.guild_id: Optional[int] = (
-            int(data["guild_id"]) if "guild_id" in data else None
-        )
-        self.primary_sku_id: Optional[int] = (
-            int(data["primary_sku_id"]) if "primary_sku_id" in data else None
-        )
-        self.slug: Optional[str] = data["slug"] if "slug" in data else None
-        self.cover_image: Optional[str] = (
-            data["cover_image"] if "cover_image" in data else None
-        )
-        self.flags: Optional[ApplicationFlags] = (
-            ApplicationFlags(data["flags"]) if "flags" in data else None
-        )  # TODO: Values
-        self.tags: Optional[List[str]] = (
-            data["tags"] if "tags" in data else None
-        )
-        self.install_params: Optional[InstallParams] = (
-            InstallParams(data["install_params"])
-            if "install_params" in data
-            else None
-        )
-        self.custom_install_url: Optional[str] = (
-            data["custom_install_url"] if "custom_install_url" in data else None
-        )
-        self.role_connections_verification_url: Optional[str] = (
-            data["role_connections_verification_url"]
-            if "role_connections_verification_url" in data
-            else None
+        self.verify_key = data["verify_key"]
+        self.team = instance_or_none(Team, data.get("team"))
+        self.guild_id = int_or_none(data.get("guild_id"))
+        self.primary_sku_id = int_or_none(data.get("primary_sku_id"))
+        self.slug = data.get("slug")
+        self.cover_image = data.get("cover_image")
+        self.flags = instance_or_none(ApplicationFlags, data.get("flags"))
+        # TODO: Values
+        self.tags = data.get("tags")
+        self.install_params = instance_or_none(
+            InstallParams, data.get("install_params")
+        )
+        self.custom_install_url = data.get("custom_install_url")
+        self.role_conn_verification_url = data.get(
```

Comment:
I don't like this abbreviation, all the names should be in full as documented.

Reply:
Well it you got any idea, but that full name is quite long tbh

Reply:
Well there surely a better name than `role_connection_verification_url` which is 32 characters long

Reply:
But it's what Discord document it as, I don't want to go away from their names as it breaks consistency